src/view/actions/sendEvent.jsx

```diff
@@ -18,85 +18,40 @@ import {
   Picker,
   TextField,
   Checkbox
-} from "../components/hookFormReactSpectrum";
+} from "../components/formikReactSpectrum";
```

Comment:
Should this be formikReactSpectrum3

Reply:
I didn't put `3` in there because there was no naming collision, but it might be better to do so for clarity until we have react-spectrum 2 cleared out.